repository/l.json

```diff
@@ -1741,6 +1741,10 @@
 			"name": "LSP-pyright",
 			"details": "https://github.com/sublimelsp/LSP-pyright",
 			"releases": [
+				{
+					"sublime_text": "3154 - 4069",
```

Comment:
Up to `3999` is enough. :)

Reply:
What about 4000-4069 then?

Reply:
`"sublime_text": "3154 - 4069",` is taken from `LSP` actually.

Reply:
Never mind. Should be fine. :)